src/main/resources/import.sql

```diff
@@ -0,0 +1,40 @@
+-- --   ----------------------------------------------------------------------------------------------
+-- --  Fill projects table
+-- --   ----------------------------------------------------------------------------------------------
+INSERT INTO startup_platform.projects (id, name, owner_id, category_id, cost, date_added, date_start, min_invest, description, goal, stage) VALUES(1, 'Raccoon.World - gadgets for hands', 1, 1, 400000, {ts '2017-05-20 14:30:30.069'}, {ts '2017-05-20 15:30:30.069'}, 50000, 'We are building Virtual Insurance Agents (chatbots) to help insurers to engage their customers via Facebook Messengers.', 'goal', 'start'),(2, 'Integrating Virtuality into Reality, Game', 1, 1, 150000, {ts '2017-05-20 14:30:30.069'},{ts  '2017-05-20 15:30:30.069'}, 50000,'Mobile MMO RPG (midcore), with Step-by-step combat system, and focus on multiplayer real-time PvP.', 'goal','start'),(3, 'Wood Pellets Storage', 1, 2, '1800000', {ts '2017-05-20 14:30:30.069'}, {ts '2017-05-20 15:30:30.069'}, 60000, 'European winter storage-- All math and figures are taken from personal experience obtained during last year- Pellets sales market- Europe', 'goal','start'),(4, 'Virtual Insurance Agents (chatbots) for insurers', 1, 2, 400000, {ts '2017-05-20 14:30:30.069'}, {ts '2017-05-20 15:30:30.069'}, 20000,'We are building Virtual Insurance Agents (chatbots) to help insurers to engage their customers via Facebook Messengers.','goal', 'middle'),(5, 'Appio', 1, 3, 500000, {ts '2017-05-20 14:30:30.069'}, {ts '2017-05-20 15:30:30.069'}, 25000,'We transformed huge, long, expensive and crowded of people process to really simple idea, full-stack turnkey solution less than a week and almost free.','goal', 'middle'),(6, 'Coliving Hub', 2, 1, 450000, {ts '2017-05-20 14:30:30.069'}, {ts '2017-05-20 15:30:30.069'}, 100000,'Affordable housing in San Francisco for digital nomads', 'goal', 'middle'),(7, 'Service for converting audio format to text format', 2, 1, 300000, {ts '2017-05-20 14:30:30.069'}, {ts '2017-05-20 15:30:30.069'}, 30000,'Transcribe multiple voices from any meeting, interview, conference or other voice negotiations. Ability to complete a text search from transcribed files.','goal', 'middle'),(8, 'global marketplace for tutoring', 2, 2, 5000000, {ts '2017-05-20 14:30:30.069'}, {ts '2017-05-20 15:30:30.069'}, 500000,'a global marketplace for tutoring that is launched in 5 countries.', 'goal', 'finish'),(9, 'Kevuru Games', 2, 1, 100000, {ts '2017-05-20 14:30:30.069'}, {ts '2017-05-20 15:30:30.069'}, 10000,'Mobile games development with focus on strategy, match 3 and slots games categories', 'goal', 'finish'),(10, 'ebooks', 2, 2, 5000, {ts '2017-05-20 14:30:30.069'}, {ts '2017-05-20 15:30:30.069'}, 2500,'Production of the cheap eBook and introduction them to the markets as a substitute of classic paper books.','goal', 'finish');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal 'start' 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal '2017-05-20 14:30:30.069' 10 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal 'finish' 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal 'middle' 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal '2017-05-20 15:30:30.069' 20 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)